index.html

```diff
@@ -220,7 +220,7 @@ <h1>Contact me</h1>
                     <textarea maxlength="500" id="msg" class="contact_form_message" name="user_message" placeholder="Enter your message here..." required></textarea>
                 </li>
                 <li>
-                    <button class="contact-form-submit" type="submit" value="Validate">Get in Touch</button>
+                    <button onclick="addToStorage()" class="contact-form-submit" type="submit" value="Validate">Get in Touch</button>
```

Comment:
- [ ] _If at the previous visit to the page, in the same browser, no input was entered in the form then the form inputs should remain empty._ Currently, when I first loaded your page there was input there, that should not be the case
![Screenshot from 2021-10-15 23-24-06](https://user-images.githubusercontent.com/62705947/137555226-b79e3b51-dff3-47a5-8577-8bc020c28bab.png)

- [ ] _After filling the form, and not submitting it, the entered values should be saved in the browser._ Currently whatever input I add the form when I refresh it returns the first thing it was added when I first entered the page not the thing I added
- [ ] When the user loads the page, the code checks if there are any saved data in the local storage and if so the input fields are pre-filled with the data